apps/frontend/src/components/ViewFormsList/ViewFormsList.tsx

```diff
@@ -87,12 +97,23 @@ export default function ViewFormsList() {
   useEffect(() => {
     getForms();
   }, []);
-
+  // event handler for all the checkboozes to a empty list until the boxes are checked then the list appends the checked boxes
```

Comment:
These comments seem like they're in the wrong place? Not really sure what they mean